git/homework.py

```diff
@@ -16,6 +17,9 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     In another case should return False
     """
     pass
+    return True if first == second else False
```

Comment:
you do not need this check, operators like `is, in, ==` already return bool value